### PR TITLE
[Short] Small test tool's CLI parser refactor

### PR DESCRIPTION
**Pull request by pbatko-da**
CHANGELOG_BEGIN
CHANGELOG_END

### CODE REVIEW

File: ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/Cli.scala [hunk 1]
```diff
@@ -17,9 +16,11 @@ import scala.util.Try
 
 object Cli {
 
-  private def reportUsageOfDeprecatedOption[B](
-      option: String
-  ) = { (_: Any, config: B) =>
+  private val Name = "ledger-api-test-tool"
+
+  private[this] implicit val tlsVersionRead: Read[File] = Read.reads(Paths.get(_).toFile)
```
Comment (kamil-da) [inline ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/Cli.scala:21]: Shouldn't this have a more generic name? It's used for various parameters.

File: ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/Cli.scala [hunk 2]
```diff
@@ -96,32 +74,38 @@ object Cli {
...
       .optional()
       .action((v, c) => c.copy(timeoutScaleFactor = v))
       .text(
         """Scale factor for timeouts used in all test suites. Useful to tune timeouts
-              |depending on the environment and the Ledger implementation under test.
-              |Defaults to 1.0. Use numbers higher than 1.0 to make test timeouts more lax,
-              |use numbers lower than 1.0 to make test timeouts more strict.""".stripMargin
+          |depending on the environment and the Ledger implementation under test.
+          |Defaults to 1.0. Use numbers higher than 1.0 to make test timeouts more lax,
+          |use numbers lower than 1.0 to make test timeouts more strict.""".stripMargin
```
Comment (kamil-da) [inline ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/Cli.scala:108]: Where are these indent changes come from? Do you use any special formatter?

Comment (pbatko-da) [reply ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/Cli.scala:108]: No, I'm using the defaults (afaict)

File: ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/Cli.scala (hunk 1 above)
Comment (pbatko-da) [reply ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/Cli.scala]: Ok

**Review by kamil-da — APPROVED** [review]

File: ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/Cli.scala (hunk 2 above)
Comment (hubert-da) [reply ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/Cli.scala:108]: @pbatko-da please check if you have "Scalafmt" under Preferences -> Editor -> Code style -> Scala -> Formatter.

**Pull request merged**